frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -974,7 +974,10 @@ function ReaderHighlight:exportToDocument(page, item)
 end
 
 function ReaderHighlight:addNote()
-    self:handleEvent(Event:new("addNote"))
+    self.ui:handleEvent(Event:new("AddNote"))
```

Comment:
This whole business of having these events at the top doesn't seem to make much sense to me. Surely the idea is more along the lines of "hey guys, we've added a note, go wild", and not so much "hey guys, we're going to add a note but you can't do anything about it."

But I actually started writing this comment to say that you should either remove the logger.dbg call or keep 'em together.